Sources/Demo/AppDelegate.swift

```diff
@@ -9,6 +10,8 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
         _ application: UIApplication,
         didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?
     ) -> Bool {
+        // Atlantis.start(hostName: "")
+
```

Comment:
nit: delete commented out code.

Reply:
This is present to speed up my development when I need to use Atlantis (along with the `import`)